care/facility/api/serializers/patient_consultation.py

```diff
@@ -239,9 +302,50 @@ def create(self, validated_data):
         consultation.last_edited_by = self.context["request"].user
         consultation.save()
 
+        try:
+            health_details_data = self.context["request"].data[
+                "new_health_details"
+            ]
+            vaccination_history = health_details_data.pop(
+                "vaccination_history", []
+            )
+
+            health_details = PatientHealthDetails(
+                patient=consultation.patient,
```

Comment:
All of these should be authorised

Reply:
@vigneshhari The authorization checks are already present in the consultation API for consultation. Will the auth check for `health_details` be different from the consultation?